frontend/components/FilterSearch.js

```diff
@@ -0,0 +1,153 @@
+import { useState, useEffect } from 'react'
+import s from 'styled-components'
+import Select from 'react-select/async'
+import fuzzysort from 'fuzzysort'
+
+import { Icon } from './common'
+import { FilterHeader } from './DropdownFilter'
+import {
+  mediaMaxWidth,
+  MD,
+  ANIMATION_DURATION,
+} from '../constants/measurements'
+import {
+  ALLBIRDS_GRAY,
+  MEDIUM_GRAY,
+  CLUBS_RED,
+  CLUBS_GREY,
+  FOCUS_GRAY,
+  BORDER,
+  WHITE,
+} from '../constants/colors'
+
+const SearchWrapper = s.div`
+  margin-bottom: 30px;
+  max-height: 0;
+  opacity: 0;
+  overflow: visible;
+  transition: all ${ANIMATION_DURATION}ms ease;
+
+  ${({ active }) => active && 'max-height: 150vh; opacity: 1;'}
```

Comment:
Why 150vh?

Reply:
changed to 100%, needed some value to override max-height: 0 set above when active is true